### PR TITLE
Change repositories list to the actual one.

### PR DESCRIPTION
**Pull request by DexterHD**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-414/

File: README.md [hunk 1]
```diff
@@ -1,25 +1,46 @@
...
+### PMM Server
+
+* [percona/pmm-managed](https://github.com/percona/pmm-managed/tree/PMM-2.0)
+* [percona/qan-api2](https://github.com/percona/qan-api2/tree/master)
+* [percona/qan-app](https://github.com/percona/qan-app/tree/PMM-2.0)
+* [percona/pmm-update](https://github.com/percona/pmm-update/tree/PMM-2.0)
+* [percona/percona-toolkit](https://github.com/percona/percona-toolkit/tree/3.0)
+* [percona/grafana-dashboards](https://github.com/percona/grafana-dashboards/tree/PMM-2.0)
+* [percona/pmm-manage](https://github.com/percona/pmm-manage/tree/PMM-2.0)
+* [percona/pid-watchdog](https://github.com/percona/pid-watchdog/tree/master)
```
Comment (AlekSi) [inline README.md]: The last two will not be used by GA.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,46 @@
...
+
+### PMM Client
+
+* [percona/pmm-agent](https://github.com/percona/pmm-agent/tree/master)
+* [percona/pmm-admin](https://github.com/percona/pmm-admin/tree/master)
+* [percona/node_exporter](https://github.com/percona/node_exporter)
```
Comment (AlekSi) [inline README.md]: There you switched from links to branches to link to repositories. Why?

File: README.md [hunk 3]
```diff
@@ -1,25 +1,46 @@
...
+* [percona/pmm-server](https://github.com/percona/pmm-server/tree/PMM-2.0)
+* [percona/pmm-server-packaging](https://github.com/percona/pmm-server-packaging/tree/PMM-2.0)
+* [Percona-Lab/pmm-submodules](https://github.com/Percona-Lab/pmm-submodules/tree/PMM-2.0)
+
+### QA and Testing
+
+* [percona/pmm-qa](https://github.com/percona/pmm-qa/tree/PMM-2.0)
+* [Percona-Lab/pmm-api-tests](https://github.com/Percona-Lab/pmm-api-tests/tree/master)
+
+
```
Comment (AlekSi) [inline README.md]: One trailing empty lines please.

File: README.md (hunk 2 above)
Comment (rnovikovP) [reply README.md]: Let's use branches only if this is not  a `default`

File: README.md (hunk 3 above)
Comment (rnovikovP) [reply README.md]: is it better now?

**Comment by rnovikovP** [comment]
@AlekSi  I see this is as DEV team to have simple view on What is used right  now  (yes, this is covered in Submodules) but this gives simpler way to navigate in the product repos

**Comment by AlekSi** [comment]
> I see this is as DEV team to have simple view on What is used right now

Then I think we should add:
* https://github.com/Percona-QA/percona-qa
* https://github.com/Percona-Lab/jenkins-pipelines
* https://github.com/Percona-Lab/percona-images

And maybe something else.

**Comment by AlekSi** [comment]
> yes, this is covered in Submodules

Then this README might belong to submodules repo?

In general, all our READMEs do not have a clear target audience: customers? users? external contributors? people trying to build PMM themselves? bystanders? competitors?

**Comment by rnovikovP** [comment]
@puneet0191 , @Nailya  do we use https://github.com/Percona-QA/percona-qa or it's 100%  replaced by  https://github.com/percona/pmm-qa/tree/PMM-2.0 ?

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-414/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-414/

**Comment by puneet0191** [comment]
@rnovikovP we have a common script being used my multiple teams, which is to fetch download url of the product, that is the only file we use from percona-qa repo, the rest have been moved to pmm-qa repo, including jobs etc.

**Comment by rnovikovP** [comment]
> all our READMEs do not have a clear target audience

Will be defined  better in https://jira.percona.com/browse/PMM-4567

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-414/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-414/

**Pull request merged**